pages/common/prettier.md

```diff
@@ -22,3 +22,7 @@
 - Format files or directories recursively using single quotes and no trailing commas:
 
 `prettier --single-quote --trailing-comma {{none}} --write {{path/to/file_or_directory}}`
+
+- Format JavaScript and TypeScript files recursively, replacing the original:
+
+`prettier --write "**/*.{js,jsx,ts,tsx}"`
```

Comment:
```suggestion
`prettier --write "**/*.{{js,jsx,ts,tsx}}"`
```

Reply:
`prettier --write "**/*.{js,jsx,ts,tsx}"` is whole command. 
if you highlight it, you can highlight `prettier --write "**/*.{{{js,jsx,ts,tsx}}}"` like this